### PR TITLE
Create ui-c8y-1021-11-1-bulk-operation-scheduling-delay-in-milliseconds-not-working-7607-graft-release.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [DM-4183](https://cumulocity.atlassian.net/browse/DM-4183)
Original PR: [7677](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7677)

[DM-4183]: https://cumulocity.atlassian.net/browse/DM-4183?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1021-11-1-bulk-operation-scheduling-delay-in-milliseconds-not-working-7607-graft-release.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: bulk operation scheduling delay in milliseconds not working (#7607) [GRAFT][release/cd] (#7677)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1021-11-1-bulk-operation-scheduling-delay-in-milliseconds-not-working-7607-graft-release.md]: The Claude AI suggested to use the following title:
```suggestion
title: Bulk operation scheduling delay in milliseconds is now working
```

File: content/change-logs/device-management/ui-c8y-1021-11-1-bulk-operation-scheduling-delay-in-milliseconds-not-working-7607-graft-release.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-4183
+version: 1021.11.1
+---
+bulk operation scheduling delay in milliseconds not working (#7607) [GRAFT][release/cd] (#7677)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1021-11-1-bulk-operation-scheduling-delay-in-milliseconds-not-working-7607-graft-release.md]: The Claude AI suggested to use the following body:
```suggestion
In the Cumulocity IoT platform, bulk operations allow performing actions on multiple devices simultaneously. However, the scheduling delay in milliseconds for these operations was not working as expected. This issue has now been resolved. With this fix, bulk operations can be scheduled with a delay specified in milliseconds, providing more precise control over when the operations are executed on the target devices. This improvement ensures that bulk operations are carried out according to the specified delay, enhancing the reliability and efficiency of device management tasks.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@dominikschmitz Please review and approve. For some reasons I cannot add you as reviewer.

File: content/change-logs/device-management/ui-c8y-1021-11-1-bulk-operation-scheduling-delay-in-milliseconds-not-working-7607-graft-release.md (hunk 2 above)
Comment (dosc-c8y) [reply content/change-logs/device-management/ui-c8y-1021-11-1-bulk-operation-scheduling-delay-in-milliseconds-not-working-7607-graft-release.md]: Maybe shorter?

Scheduling bulk operations with delays in milliseconds has been fixed. Operations can now be scheduled with precise delays, improving the reliability and efficiency of device management tasks.

File: content/change-logs/device-management/ui-c8y-1021-11-1-bulk-operation-scheduling-delay-in-milliseconds-not-working-7607-graft-release.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Bulk operation scheduling delay in milliseconds is now working
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1021-11-1-bulk-operation-scheduling-delay-in-milliseconds-not-working-7607-graft-release.md]:
```suggestion
title: Bulk operation scheduling delay in milliseconds now properly
```

File: content/change-logs/device-management/ui-c8y-1021-11-1-bulk-operation-scheduling-delay-in-milliseconds-not-working-7607-graft-release.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Bulk operation scheduling delay in milliseconds now properly
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1021-11-1-bulk-operation-scheduling-delay-in-milliseconds-not-working-7607-graft-release.md]:
```suggestion
title: Bulk operation scheduling delay in milliseconds now works properly
```

File: content/change-logs/device-management/ui-c8y-1021-11-1-bulk-operation-scheduling-delay-in-milliseconds-not-working-7607-graft-release.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-4183
+version: 1021.11.1
+---
+In the Cumulocity IoT platform, bulk operations allow performing actions on multiple devices simultaneously. However, the scheduling delay in milliseconds for these operations was not working as expected. This issue has now been resolved. With this fix, bulk operations can be scheduled with a delay specified in milliseconds, providing more precise control over when the operations are executed on the target devices. This improvement ensures that bulk operations are carried out according to the specified delay, enhancing the reliability and efficiency of device management tasks.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1021-11-1-bulk-operation-scheduling-delay-in-milliseconds-not-working-7607-graft-release.md]:
```suggestion
In the {{< product-c8y-iot >}} platform, bulk operations allow performing actions on multiple devices simultaneously. However, scheduling these operations with a delay in milliseconds was not working as expected. This issue has now been resolved. With this fix, bulk operations can now be scheduled with precise delays specified in milliseconds, enhancing the reliability and efficiency of device management tasks.
```

File: content/change-logs/device-management/ui-c8y-1021-11-1-bulk-operation-scheduling-delay-in-milliseconds-not-working-7607-graft-release.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Bulk operation scheduling delay in milliseconds now works properly
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1021-11-1-bulk-operation-scheduling-delay-in-milliseconds-not-working-7607-graft-release.md]:
```suggestion
title: Scheduling bulk operations with a precise delay now works properly
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@dosc-c8y Please approve

**Review by dosc-c8y — APPROVED** [review]

**Pull request merged**